__tests__/app.test.js

```diff
@@ -30,8 +33,73 @@ describe("GET api/", () => {
     return request(app)
       .get("/api")
       .expect(200)
-      .then((response) => {
-        expect(response.body).toEqual(expect.any(Object))
+      .then((readApi) => {
+        expect(readApi.body).toEqual(endpointsFile)
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with a 200 status code and gets a single article by its id", () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then((response) => {
+      const article = response.body.article
+    expect(article).toMatchObject({
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      votes: 100,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+    })
+    })
+  })
+  test("200: responds with 200 to get any article by its id", () => {
+    const article_id = 2
```

Comment:
I think this test is a bit overkill as we've already showed the endpoint works with the first test. This one is just a duplication but for a different endpoint, so can be removed 👍